package.json

```diff
@@ -58,6 +58,7 @@
     "cross-env": "^5.0.5",
     "eslint": "^6.8.0",
     "esprima": "^4.0.0",
+    "expect": "25.2.7",
```

Comment:
Why is this an exact version and not a `^` version range like the others?

Reply:
Primarily because we got caught out running `typescript: '^3.6'` and CI installed a different version than I was running locally. Therefore I'd rather pin things so CI has the same versions.

Alternatively we could commit the `package-lock.json` and that way everything would run the same version?

Reply:
This is already a problem (or non-problem, depending on your POV) for all the other dependencies as well. I'd prefer not deviating from the way we version other dependencies. Doing things inconsistently is confusing. Let's stick with`^` for now; when an `expect` update comes out that doesn't respect semver properly and breaks something, CI will tell us about it. WDYT?

Reply:
👍 done!